packages/clerk-js/src/core/clerk.ts

```diff
@@ -451,6 +451,8 @@ export default class Clerk implements ClerkInterface {
       params.afterSignUpUrl || params.redirectUrl || displayConfig.afterSignUpUrl,
     );
 
+    const navigateToContinueSignUp = makeNavigate(displayConfig.signUpUrl + '#/continue');
```

Comment:
One more hard link joins the 'routes-we-need-to-refactor' gang 😄 

PS: Not related to this PR